### PR TITLE
Display Greenwave details if available

### PR DESCRIPTION
**Pull request by mgrabovsky**
This is useful mainly for the `invalid-gating-yaml` virtual test.

Reference: OSCI-4453

### CODE REVIEW

**Comment by mgrabovsky** [comment]
Here's a quick example from the build mentioned in OSCI-4453:
image

**Comment by mgrabovsky** [comment]
@Andrei-Stepanov Can you please review?

File: src/artifact.ts [hunk 1]
```diff
@@ -192,8 +192,9 @@ export type GreenwaveRequirementType = {
     subject_type: string;
     /* python2-flask-1.0.2-1.rawhide */
     subject_identifier: string;
-    result_id: number;
-    error_reason: string;
+    result_id?: number;
```
Comment (Andrei-Stepanov) [inline src/artifact.ts:195]: @mgrabovsky do you have an example when there this entry in undefied?
Do you have buid NVR?

Comment (mgrabovsky) [reply src/artifact.ts:195]: See pcm-202211-0.el9, it's linked in the ticket mentioned above.

Comment (Andrei-Stepanov) [reply src/artifact.ts:195]: Ok

**Pull request merged**